bav-ipv-stub/get-ssm-test-params.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
```

Comment:
Do we actually need this file I don't think we ever used it in F2F/CIC?

Reply:
Deleted.